l10n/de.json

```diff
@@ -29,6 +29,7 @@
     "Attribute to map the quota to." : "Attribut, dem das Kontingent zugeordnet werden soll.",
     "Attribute to map the users groups to." : "Attribut, dem die Gruppen des Benutzers zugeordnet werden sollen.",
     "Attribute to map the users home to." : "Attribut dem das zu Hause des Benutzers zugeordnet werden soll.",
+    "Attribute to map the users avatar to." : "Attribut, dem das das Profilbild des Benutzers zugeordnet werden soll.",
```

Comment:
Please don't change any translations here directly – they are being done on transifex, changes would be overwritten.